src/pages/settings/EditPasswordScreen.js

```diff
@@ -0,0 +1,149 @@
+    ...withCurrentUserPersonalDetailsDefaultProps,
+};
+
+class EditPasswordScreen extends Component {
+    constructor(props) {
+        super(props);
+        this.submit = this.submit.bind(this);
+        this.validate = this.validate.bind(this);
+
+        this.errorKeysMap = {
+            currentPassword: 'passwordPage.errors.currentPassword',
+            newPasswordSameAsOld: 'passwordPage.errors.newPasswordSameAsOld',
+            newPassword: 'passwordPage.errors.newPassword',
+        };
+    }
+
+    /**
+     * @param {Object} values - form input values passed by the Form component
+     * @returns {Object}
+     */
+    validate(values) {
+        const errors = {};
+        if (!values.currentPassword) {
+            errors.currentPassword = this.props.translate(this.errorKeysMap.currentPassword);
+        }
+
+        if (!values.newPassword || !ValidationUtils.isValidPassword(values.newPassword)) {
+            errors.newPassword = this.props.translate(this.errorKeysMap.newPassword);
+        }
+
+        if (values.currentPassword && values.newPassword && _.isEqual(values.currentPassword, values.newPassword)) {
+            errors.newPasswordSameAsOld = this.props.translate(this.errorKeysMap.newPasswordSameAsOld);
+        }
+        return errors;
+    }
+
+    /**
+     * @param {Object} values - form input values passed by the Form component
+     */
+    submit(values) {
+        User.updatePassword(values.currentPassword, values.newPassword);
+    }
+
+    render() {
+        return (
+            <ScrollView
+                style={styles.flex1}
+                contentContainerStyle={styles.p5}
+
+                // Allow the user to click show password while password input is focused.
+                // eslint-disable-next-line react/jsx-props-no-multi-spaces
+                keyboardShouldPersistTaps="always"
+            >
+                <Text style={[styles.mb6]}>
+                    {this.props.translate('passwordPage.changingYourPasswordPrompt')}
+                </Text>
+                <Form
+                    formID={ONYXKEYS.FORMS.EDIT_PASSWORD_FORM}
+                    validate={this.validate}
+                    onSubmit={this.submit}
+                    submitButtonText={this.props.translate('common.save')}
+                >
+                    <View style={styles.mb6}>
+                        <TextInput
+                            label={`${this.props.translate('passwordPage.currentPassword')}*`}
+                            secureTextEntry
+                            autoCompleteType="password"
+                            textContentType="password"
+                            returnKeyType="done"
+                            inputID="currentPassword"
+                        />
+                    </View>
+                    <View style={styles.mb6}>
+                        <TextInput
+                            label={`${this.props.translate('passwordPage.newPassword')}*`}
```

Comment:
This seems to be the only place that uses this translation key. Can we just move `*` to the translation files and simplify the code here?
```suggestion
                            label={this.props.translate('passwordPage.newPassword')}
```

Reply:
It seems to be weird when we change newPassword: 'New password*'. That makes things confused

Reply:
Hmm actually I don't think that we need `*` at all. We don't use that in any required fields.

Reply:
I think we should remove those from Password page as well